feature-tests/package.json

```diff
@@ -38,5 +38,8 @@
     "@aws-sdk/client-sqs": "^3.654.0",
     "@aws-sdk/credential-providers": "^3.654.0",
     "@aws-sdk/util-dynamodb": "^3.656.0"
+  },
+  "resolutions": {
+    "form-data": "4.0.4"
```

Comment:
Update supertest package instead of this. This should be a workaround for `pact` libraries only